lib/tagnews/geoloc/tag.py

```diff
@@ -27,9 +29,18 @@
 
 
 def post_process(geostring):
-    # TODO
-    geostring += ' Chicago, Illinois'
-    geostring.replace('block of ', '')
+    assert type(
```

Comment:
I don't think we need this check anymore, also `addr_list` is not a variable in this function